database/tables.sql

```diff
@@ -1,84 +1,88 @@
 CREATE TABLE USUARIO (
-    EMAIL VARCHAR(50) PRIMARY KEY,
-    TIPO VARCHAR(20)
+    EMAIL VARCHAR(50) PRIMARY KEY NOT NULL,
+    TIPO VARCHAR(20) NOT NULL
 );
 
+
 CREATE TABLE SEGURO (
-    ID SERIAL PRIMARY KEY,
-    NOMBRE VARCHAR(50),
-    ESTADO_ACTIVO BOOLEAN
+    ID SERIAL PRIMARY KEY NOT NULL,
+    NOMBRE VARCHAR(50) NOT NULL,
+    ESTADO_ACTIVO BOOLEAN NOT NULL
 );
 
 CREATE TABLE PACIENTE (
     ID SERIAL PRIMARY KEY,
     CUI VARCHAR(24),
-    CORREO VARCHAR(50),
-    ES_MUJER BOOLEAN,
-    NOMBRES VARCHAR(24),
-    APELLIDOS VARCHAR(24),
-    NOMBRE_CONTACTO1 VARCHAR(50),
-    PARENTESCO_CONTACTO1 VARCHAR(24),
-    TELEFONO_CONTACTO1 VARCHAR(20),
+    CORREO VARCHAR(50) NOT NULL,
+    ES_MUJER BOOLEAN NOT NULL,
+    NOMBRES VARCHAR(50) NOT NULL,
+    APELLIDOS VARCHAR(50) NOT NULL,
+    NOMBRE_CONTACTO1 VARCHAR(50) NOT NULL,
+    PARENTESCO_CONTACTO1 VARCHAR(24) NOT NULL,
+    TELEFONO_CONTACTO1 VARCHAR(20) NOT NULL,
     NOMBRE_CONTACTO2 VARCHAR(50),
     PARENTESCO_CONTACTO2 VARCHAR(24),
     TELEFONO_CONTACTO2 VARCHAR(20),
-    TIPO_SANGRE VARCHAR(3),
-    DIRECCION VARCHAR(100),
-    ID_SEGURO INTEGER,
-    FECHA_NACIMIENTO DATE,
-    TELEFONO VARCHAR(20),
+    TIPO_SANGRE VARCHAR(3) NOT NULL,
+    DIRECCION VARCHAR(100) NOT NULL,
+    ID_SEGURO INTEGER NOT NULL,
+    FECHA_NACIMIENTO DATE NOT NULL,
+    TELEFONO VARCHAR(20) NOT NULL,
     FOREIGN KEY (ID_SEGURO) REFERENCES SEGURO (ID)
 );
 
+
+
 CREATE TABLE ESTUDIANTE (
-    CARNET VARCHAR(20),
-    CARRERA VARCHAR(50),
+    CARNET VARCHAR(20) NOT NULL,
+    CARRERA VARCHAR(50) NOT NULL,
     ID_PACIENTE INTEGER NOT NULL,
     PRIMARY KEY (CARNET),
     FOREIGN KEY (ID_PACIENTE) REFERENCES PACIENTE (ID)
 );
 
 CREATE TABLE COLABORADOR (
-    CODIGO VARCHAR(20) PRIMARY KEY,
-    AREA VARCHAR(50),
+    CODIGO VARCHAR(20) PRIMARY KEY NOT NULL,
+    AREA VARCHAR(50) NOT NULL,
     ID_PACIENTE INTEGER NOT NULL,
     FOREIGN KEY (ID_PACIENTE) REFERENCES PACIENTE (ID)
 );
 
 CREATE TABLE CONSULTA (
-    ID SERIAL PRIMARY KEY,
+    ID SERIAL PRIMARY KEY NOT NULL,
     ID_PACIENTE INTEGER NOT NULL,
-    FECHA TIMESTAMP,
-    MOTIVO VARCHAR(100),
-    FRECUENCIA_RESPIRATORIA VARCHAR(10),
-    TEMPERATURA DOUBLE PRECISION,
-    SATURACION_OXIGENO DOUBLE PRECISION,
-    GLUCOMETRIA DOUBLE PRECISION,
-    FRECUENCIA_CARDIACA DOUBLE PRECISION,
-    PRESION_SISTOLICA INTEGER,
-    PRESION_DIASTOLICA INTEGER,
-    EVALUADOR VARCHAR(50),
-    FOREIGN KEY (ID_PACIENTE) REFERENCES PACIENTE (ID)
+    FECHA TIMESTAMP NOT NULL,
+    MOTIVO TEXT NOT NULL,
+    FRECUENCIA_RESPIRATORIA VARCHAR(10) NOT NULL,
+    TEMPERATURA DOUBLE PRECISION NOT NULL,
+    SATURACION_OXIGENO DOUBLE PRECISION NOT NULL,
+    GLUCOMETRIA DOUBLE PRECISION NOT NULL,
+    FRECUENCIA_CARDIACA DOUBLE PRECISION NOT NULL,
+    PRESION_SISTOLICA INTEGER NOT NULL,
+    PRESION_DIASTOLICA INTEGER NOT NULL,
```

Comment:
Estos datos no siempre los ingresan todos, así que puede que algunos sean null y otros no. Quitales los Not Null